internal/rego_default/rego_default_v1.go

```diff
@@ -0,0 +1,10 @@
+// Copyright 2024 The OPA Authors.  All rights reserved.
+// Use of this source code is governed by an Apache2
+// license that can be found in the LICENSE file.
+
+//go:build !opa_rego_v0
+
+package rego_default
+
+// DefaultRegoVersion is the default Rego version (v1).
+var DefaultRegoVersion = 3 // enum value for ast.RegoV1
```

Comment:
Because of circular import issues, we can't assign `ast.RegoV1` directly.

Alternatively, we could set this to the manifest int representation (0 for v0, 1 for v1), but that would impose a slight conversion overhead on the read side.